editor/src/core/model/conditionals.ts

```diff
@@ -167,3 +167,19 @@ export function maybeBranchConditionalCase(
     return null
   }
 }
+
+export function getConditionalActiveCase(
```

Comment:
Under what conditions is this different to `getConditionalCase`?

Reply:
this is a method to get the active case of a conditional, while `getConditionalCase` returns the case of a branch inside a conditional :)

Reply:
Oh my word, `getConditionalCase` definitely needs renaming then 😩 (not in this PR though)

Reply:
yeah it's growing all over the place